assets/src/edit-story/units/dimensions.js

```diff
@@ -20,13 +20,11 @@
 import {
   PAGE_WIDTH,
   PAGE_HEIGHT,
-  FULLBLEED_RATIO,
   DEFAULT_EM,
+  FULLBLEED_HEIGHT,
+  DANGER_ZONE_HEIGHT,
 } from '../constants';
 
-export const FULLBLEED_HEIGHT = PAGE_WIDTH / FULLBLEED_RATIO;
-export const DANGER_ZONE_HEIGHT = (FULLBLEED_HEIGHT - PAGE_HEIGHT) / 2;
```

Comment:
`FULLBLEED_HEIGHT` and `DANGER_ZONE_HEIGHT` are currently defined in many places throughout `assets/src/animation` and `assets/src/edit-story`.

Let's keep a single definition here in `dimensions.js` so that we can move this all to the `units` package.

Here's how I did this in the experimental `try/next-level-packages` branch:

https://github.com/google/web-stories-wp/blob/try/next-level-packages/packages/units/src/constants.js